### PR TITLE
Some damage de-escalation stuff.

### PR DESCRIPTION
**Pull request by Isy232**
##  About The Pull Request

I'd like to start off by addressing that this is what I think is good for a start to de-escalation. I am putting nerfs on the things everyone seems to hate the most in this PR. I am not doing any nuclear level changes, I mostly changed things around 10%. Things may need additional or less nerfing or there may be things that need it that I have not yet addressed in this PR.

I acknowledge that recently I have been playing xeno more than marine, I may have some biases, DM @Isy232 on discord if you want to discuss some change I made or want to recommend one.

I have heard of others planning de-escalation changes, so far there has been no PR.
This is just as a start, I am not saying I think this is the final solution to the issue just a start.

Notes:
- Tackle damage is pain damage from throttling (disarm intent).
- Melee damage is brute damage from slashing (harm intent).
- Not all the numbers are nice but if we just go by 5's it does not allow for much precision.
- I nerfed almost every single gun but most of the guns have small nerfs.
- I am not even sure every bullet type I changed is actually used by guns in the game, I did all of the ones that sounded relevant or that I recognized.
- Ravager is not included here because it got a separate PR which I think everyone agrees was needed. Further ravager changes should probably go through that PR to keep conflicts away.
- This does not need to be the only damage de-escalation PR, I am just throwing some numbers out there because I am getting frustrated with hearing the same arguments constantly in balance and that no working PR has been made yet.

Big Nerfs List:
Crusher Tackle Damage 55/60/65/70 => 35/35/40/45
Queen Melee Damage 50/55/60/65 => 45/50/55/60
Queen Tackle Damage 55/60/65/70 => 45/50/55/60
AP Pistol Bullet Damage 25 => 22
Heavy Pistol Bullet Damage 40 => 35
Incendiary Pistol Bullet Damage 10 => 20 
Squash Pistol Bullet Damage 35 => 32
Rifle Bullet Damage 40 => 35
AP Rifle Bullet Damage 30 => 26
Heavy Rifle Bullet Damage 30 => 26
Shotgun Slug Damage 70 => 62
Shotgun Incendiary Slug Damage 50 => 40
Shotgun Flechette Damage 35 => 30
Shotgun Buckshot Damage 70 => 62
Shotgun Additional Buckshot Damage 50 => 45
Incendiary Sniper Bullet Damage 50 => 45
Smartgun Bullet Damage 30 => 23
Smartgun Bullet Penetration 5 => 10
Minigun Bullet Damage 50 => 42
Military Laser Bolt Damage 25 => 22
Overcharged Laser Bolt Damage 46 => 42
Laser Gun Heavy Laser Damage 46 => 42
Additional Laser Bolt Damage 40 => 35
Light Neurotoxin Spit Reagent Injected 6.44/7/7.28 => 6.2/6.8/7
Heavy Neurotoxin Spit Regent Injected 9.2/10/10.4 => 9/9.8/10 
Light Acid Spit Damage 20 => 18
Medium Acid Spit Damage 25 => 20
Heavy Acid Spit Damage 30 => 23
Some warrior melee nerfs.
Some slight defender tackle nerfs,
Some defiler melee nerfs.
Some bull tackle nerfs.
Heavy nerfs to ravager.
Some runner tackle damage nerfs

## Why It's Good For The Game

I think nearly everyone agrees that the current damage values are too high all around.
Additionally some things are just plain ridiculous (see crushers previous tackle damage, it was higher than the queens).

## Changelog
:cl:
balance: Crusher Tackle Damage down to a reasonable level.
balance: Most pistol bullet types down slightlyn
balance: Rifles damage down a bit.
balance: Shotgun damage down a bit.
balance: Incendiary Sniper Bullet Damage down a bit.
balance: Smartgun damage down, penetration up.
balance: Minigun Bullet Damage slightly decreased.
balance: Laser Gun Damage Down Very Slightly.
balance: Nerfs acid spits damage by quite a bit.
balance: Nerfs rav damage by a bit more.
balance: Nerfs warrior damage a bit.
balance: Nerfs defiler neuroclaw.
balance: Nerfs quite a few castes having too high tackle damage in spots.
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Put the exact value changes on the body of the PR, and use the changelog to give only a summary.

**Comment by Pariah919** [comment]
I dislike these changes personally for a few reasons.

A. It only nerfs specific castes rather than ALL melee damage. Which in general can be argued to be high overall for certain castes.

B. The damage of certain things that Isy has nerfed is perfectly fine but just has too much AP such as the lasgun normal bolt and overcharge (overcharge does 40 damage according to hugh which is generally fine but it needs it's AP docked to 20 rather than 40)

C. Acid damage is fine. It just needs the 1.5x burn removed before we can make a decision whether it does too much damage or not.

D. There are a few completely unnecessary nerfs to shitty non-marine weapons such as the AK and VP-70 here which nobody has complained about the damage of.

E. I don't really get the smartgun damage profile. The loss of 7 damage for 10 more AP seems like a fair tradeoff except the smartgun generally misses 1-2 shots because of scatter in my experience.

I dislike the PR in general as it only nerfs marines and the times I've played with it I can barely notice the difference between nerfed/unnerfed damage in general.

**Comment by Isy232** [comment]
Its hard to know which guns use which bullets, tell me those instances and I will fix them as they come up.

Some things that arent in the code are getting nerfed if they have similar counterparts, such as medium acid so if someone does something to add it into the game its balanced.

I added nerfs to far more xeno castes now, not just the ones people complain the most about.

**Comment by Isy232** [comment]
Un-nerfed incendiary pistol bullets by a bit.

About the topic of nerfing unused stuff if it looks similar to something in the game I nerf it similarly to what is there so if someone decided to use it in something the numbers are similar.

In the final cleanup I can undo those though if people dont want nerfs to unused assets included.

File: code/modules/mob/living/carbon/xenomorph/castes/defiler/castedatum_defiler.dm [hunk 1]
```diff
@@ -38,7 +38,7 @@
 	armor = list("melee" = 30, "bullet" = 15, "laser" = 15, "energy" = 30, "bomb" = XENO_BOMB_RESIST_0, "bio" = 30, "rad" = 30, "fire" = 30, "acid" = 30)
 
 	// *** Defiler Abilities *** //
-	neuro_claws_amount = DEFILER_CLAW_AMOUNT
```
Comment (Fikou) [inline code/modules/mob/living/carbon/xenomorph/castes/defiler/castedatum_defiler.dm]: you should change the define or remove it from the files

**Comment by Isy232** [comment]
Removed the now useless define

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**